# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../alsa-tools/alsa-tools-x86-64.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- alsa-tools-1.0.9/ac3dec/libac3/bitstream.c.orig	2005-09-08 15:43:54.000000000 -0700
+++ alsa-tools-1.0.9/ac3dec/libac3/bitstream.c	2005-09-08 15:48:41.000000000 -0700
@@ -94,7 +94,8 @@
 static inline void
 bitstream_fill_current()
 {
-	current_word = *((uint_32*)buffer_start)++;
+	current_word = *((uint_32*)buffer_start);
+	buffer_start += sizeof(uint_32);
 	current_word = swab32(current_word);
 }
 
